Add another three new maps to random room selection

Glamurio opened the pull request:
<!-- The text between the arrows are comments - they won't be visible on your PR. -->
<!-- To automatically tag this PR, add the label(s) surrounded by brackets anywhere, for example: [LABEL] -->
<!-- PRs should at least have one area (A-) label and at least one category (C-) label -->

## About the PR <!-- Describe the Pull Request here. What does it change? What other things could this impact? -->
It adds 3 new rooms, two 5x3, one 3x5. The ranch room makes for a terrible screenshot, lmao.
![image](https://user-images.githubusercontent.com/51779813/196536181-00e5eb1e-1340-4303-99dc-c3faf85adc6a.png)
![image](https://user-images.githubusercontent.com/51779813/196536199-66ba45da-21c9-4d92-a09d-02e74b515bbc.png)
![image](https://user-images.githubusercontent.com/51779813/196536569-033bb86b-f1b3-40a1-998f-da5ed5e00c38.png)

The last map is purposefully closed off in cardinal directions by breakable barricades, even though it was said to keep cardinal directions free. But if you'd like I can remove it, though I felt this was fitting and acceptable.

## Why's this needed? <!-- Describe why you think this should be added to the game. -->
Because more random rooms is always better!


## Changelog
<!-- If necessary, put your changelog entry below. Otherwise, please delete this section.
Use however you want to be credited in the changelog in place of CodeDude.
Use (*) for major changes and (+) for minor changes. For example: -->

frawhst reviewed (commented):
Either remove the changelog or be a bit more descriptive (summarize the rooms), as-is it doesn't provide much useful information

Glamurio commented:
> Only request is to change the grass/dirt on the ranchroom into plating or some other more common flooring so it looks more like a room set up by a staffie who stole ranch equipment rather than a full fledged ranch setup. Could maybe include a grass seed they could plant to turn the room into grass that way.

I've changed the flooring to be less natural, moved some stuff around and added a grass seed inside the shrub. Now it's more like a staffie built it, I think. That more to your liking? I'll make additional adjustments if need be.
![image](https://user-images.githubusercontent.com/51779813/201748529-95ef7ed3-633a-494d-a5cc-8bf9a6b35d39.png)

frawhst commented:
hey it's me Sord and i approve this PR
